cmd/config-migrate.go

```diff
@@ -434,6 +435,11 @@ func migrateV7ToV8() error {
 	} else {
 		srvConfig.Notify.Redis = cv7.Notify.Redis
 	}
+	if len(cv7.Notify.PostgreSQL) == 0 {
```

Comment:
This is dangerous and should be avoided,  implement a proper config migration. do not edit configs like this.  Migrate to v9.